core/src/main/java/org/infinispan/util/logging/Log.java

```diff
@@ -1193,16 +1198,19 @@ void preparedTxAlreadyExists(RecoveryAwareTransaction previous,
    @Message(value = "Finished rebalance for cache %s, topology %s", id = 336)
    void finishedRebalance(String cacheName, CacheTopology topology);
 
-   @Message(value = "The 'site' must be specified!", id = 337)
+   @Message(value = "Backup configuration must include a 'site'.", id = 337)
+   @Description("Caches that use cross-site replication must include a site in the configuration. Edit the cache and specify the name of the site in the backup configuration.")
    CacheConfigurationException backupMissingSite();
 
+// Pedro: are these three related to cross-site?
+
```

Comment:
@pruivo Are the three log messages below (338, 339, and 340) related to cross-site?

Reply:
yes
338 -> user wants a custom failure-policy but they did not specify the class to use
339 -> user configured `<backup-for>` but attribute `remote-cache` is missing or set to `null`.
340 -> user configured `<backup-for>` but attribute `remote-cache` or `remote-site` or both are missing.